src/lib/webext.ts

```diff
@@ -165,15 +166,31 @@ export async function openInNewTab(
             break
     }
 
-    if (kwargs.active === false) {
-        // load in background
-        return browserBg.tabs.create(options)
-    } else {
-        // load in background and then activate, per issue #1993
-        return browserBg.tabs
-            .create(options)
-            .then(newtab => browserBg.tabs.update(newtab.id, { active: true }))
+    // load in background
+    let tab = browserBg.tabs.create(options)
+
+    if (kwargs.active) {
+        // activate once tab has been opened, per issue #1993
```

Comment:
I haven't checked that this behaviour is the same - probably ought to since it's a crucial bit of code.